sdk/python/feast/loaders/ingest.py

```diff
@@ -119,10 +119,10 @@ def get_feature_row_chunks(
             Iterable list of byte encoded FeatureRow(s).
     """
 
-    pool = Pool(max_workers)
```

Comment:
Can you explain this please?

Reply:
Hi Willem, I think you commented on the remove pools part.
Previously I was trying to verify whether the multiprocessing work or not.

Now have refactored the ingestion part, so the labels don't go to the multiprocessing ingestion part

Reply:
Please check the latest change, it has been refactored. Now the ingestion part will receive the fields it needed rather than all the feature_set object field.

Previously I was trying to verify whether the multiprocessing is the one that caused the failed ingestion test because it was failing to pickle (on a side note, I think I need to set up my way of doing end-to-end test locally)